README.md

```diff
@@ -33,6 +33,25 @@ production-worthy implementation in 2021.
 Communications for Rust-Lightning and Lightning Development Kit happens through
 [LDK slack](http://lightningdevkit.org/).
 
+Crates
+-----------
+1. [lightning](./lightning)   
+A Bitcoin Lightning library in Rust.
```

Comment:
This feels strange - the whole repo is a "Bitcoin lightning library" - maybe for lightning reword it to say something about "The Core of the LDK library, implements the lightning protocol, channel state machine, and on-chain logic. Supports no-std and exposes on relatively low-level interfaces".

Reply:
Thanks, updated it.